### PR TITLE
Support running Docker logging plugin on ARM64

### PR DESCRIPTION
**Pull request by kvch**
## What does this PR do?

This PR adds support for running Dockerlogbeat on ARM machines.

## Why is it important?

Because we want to support running Beats on ARM.

## Checklist

- [x] My code follows the style guidelines of this project
~~- [ ] I have commented my code, particularly in hard-to-understand areas~~
~~- [ ] I have made corresponding changes to the documentation~~
~~- [ ] I have made corresponding change to the default configuration files~~
~~- [ ] I have added tests that prove my fix is effective or that my feature works~~
~~- [ ] I have added an entry in `CHANGELOG.next.asciidoc` or `CHANGELOG-developer.next.asciidoc`.~~

### CODE REVIEW

**Review by v1v — APPROVED** [review]

File: .ci/packaging.groovy [hunk 1]
```diff
@@ -203,6 +203,7 @@ pipeline {
                   'metricbeat',
                   'packetbeat',
                   'x-pack/auditbeat',
+                  'x-pack/dockerlogbeat',
```
Comment (fearful-symmetry) [inline .ci/packaging.groovy:206]: Do we want to add it to the packaging? Technically there is no "packaging" for this, it's just built and uploaded to docker hub.

Comment (kvch) [reply .ci/packaging.groovy:206]: It's part of AMD64 packaging jobs as well. It is nice to have this job to make sure we are not breaking anything.

**Review by fearful-symmetry — APPROVED** [review]

**Pull request merged**